src/common-elements/PrismDiv.tsx

```diff
@@ -48,7 +48,9 @@ export const PrismDiv = styled.div`
     color: hsl(30, 20%, 50%);
   }
 
+  .ellipsis,
```

Comment:
What was the previous style for `.ellipsis`.

Reply:
It inherited from rightPanel.textColor 😮, I prefer to group all delimitators (or similar) under the same style within the codeBlock property.